sympy/matrices/expressions/tests/test_matexpr.py

```diff
@@ -338,12 +338,3 @@ def test_MatrixElement_with_values():
 def test_inv():
     B = MatrixSymbol('B', 3, 3)
     assert B.inv() == B**-1
-
-def test_factor_expand():
```

Comment:
is this removing support for `expand` and `factor`?

Reply:
factor will work in my branch https://github.com/sympy/sympy/pull/15121. expand still doesn't work but it doesn't give an error either.